WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -2909,16 +2916,36 @@ static GSourceFuncs _handlerIntervention =
                     webkit_memory_pressure_settings_free(memoryPressureSettings);
                 }
 #endif
+
+#if WEBKIT_CHECK_VERSION(2, 42, 0)
+                double originStorageRatio = -1.0;    // -1.0 means WebKit will use the default quota (1GB)
+                if (_config.IndexedDBSize.IsSet() && _config.IndexedDBSize.Value() != 0) {
+                    originStorageRatio = static_cast<double>(_config.IndexedDBSize.Value());
```

Comment:
Shouldn't we check here if the value is between 0 and 100? We could also be reading an old config file where it was still set as bytes so way higher than 100? Shouldn't we report the latter situation that the meaning of the value changed and bytes is no longer supported?

Reply:
That's a good question, but I think we don't need to do that, since we are no longer supporting `IndexedDB` for previous versions of WebKit API. Is it correct @modeveci that nobody was using it this far, so we can only handle the new percentage configs? Is so, we can leave this as it is, and on top of that we can also change `IndexedDBSize` to uint8 like @MFransen69 suggested